tests/Spec.hs

```diff
@@ -2371,27 +2371,42 @@ main = hspec $ do
 
     it "correctly updates the metric for the merge train size after each event" $ do
       let
-        state = Project.emptyProjectState
+        state
```

Comment:
Does this test alone covers most (if not all) possibilities for failure we can have in the wild? It's just a metric, but it's better to ensure this validity. Manual testing is fine as well, but you didn't describe if you did some.

Reply:
The convenient thing about the implementation is that is a function of the state, not of specific events. The predicate `Pr.isInProgress` is coincidentally also used in the web interface to display the merge train. Therefore I believe the only functionality added is periodically emitting this information, not so much generating it.

That said, is there another specific test you would like to see?

Reply:
This reasoning looks sound, especially if you took inspiration of how the merge train is displayed! I have nothing to add.